dist/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
```

Comment:
I don't think this file is needed if we have `dist/` in the original `.gitignore`

Reply:
this is so the directory is in git when checking thge code out